src/util/PrimalityTest.c

```diff
@@ -1,11 +1,14 @@
 #include "util/PrimalityTest.h"
 #include "util/Random.h"
 
+static const int MIGHT_BE_PRIME = 1;
+static const int NOT_PRIME = 0;
+
 int primaltyTest_millerrabin_mpz(const mpz_srcptr p, const int repetitions)
```

Comment:
Small little thing: this function can now be static. You must've forgotten it :D

Reply:
Outdated.